Sources/MapboxMaps/Style/Style.swift

```diff
@@ -165,25 +170,16 @@ public class Style {
      - Parameter type: The type of the source
      - Returns: The fully formed `source` object of type equal to `type`.
      - Throws: StyleError or type conversion errors
+     - Note: This method is marked as experimental. Annotate the import statement
+     for `MapboxMaps` with `@_spi(Experimental)` in order to use experimental methods.
      */
-    public func _source(withId id: String, type: Source.Type) throws  -> Source {
+    @_spi(Experimental) public func source(withId id: String, type: Source.Type) throws  -> Source {
         // Get the source properties for a given identifier
         let sourceProps = try sourceProperties(for: id)
         let source = try type.init(jsonObject: sourceProps)
         return source
     }
 
-    /// Gets the value of style source property.
-    ///
-    /// - Parameters:
-    ///   - sourceId: Style source identifier.
-    ///   - property: Style source property name.
-    ///
-    /// - Returns: The value of the property in the source with sourceId.
-    public func sourceProperty(for sourceId: String, property: String) -> Any {
-        return _sourceProperty(for: sourceId, property: property).value
-    }
-
```

Comment:
Curious why this was removed

Reply:
This was removed because it seems like a [duplicate method](https://github.com/mapbox/mapbox-maps-ios/pull/680/files#diff-3c7e02fb492d91d8b149fb3a5799e4cca459c77771dba75e942fa715bb62d6c8R434-R436) . The difference seems to be that once returns a `StylePropertyValue` while the other returns the `StylePropertyValue.value`. Alternatively, we could rename the deleted method.

Reply:
I'm open to keeping this change. Since the deleted method was not experimental and the one we kept is, maybe we should make the one we kept non-experimental?

Reply:
I think this is another issue we can address when we do the full audit of what is experimental